README.md

```diff
@@ -0,0 +1,95 @@
+# Creative Commons - Chapter WordPress theme
+
+- Contributors: quentinrogers, mackaffinity, cctimidrobot, hugosolar
+- Tags: CreativeCommons
+- Requires at least: 4.5+
+- Tested up to: 5.0.3
+- Requires PHP: 7.0
+- License: GPLv2 or later
+- License URI: https://www.gnu.org/licenses/gpl-2.0.html
+
+Creative Commons Chapter site theme (including chapter sites setup,
+taxonomies, and widget code)
+
+## Development Installation
+
+This theme uses npm, gulp, and browserify to manage dependencies.
+
+Versions of node and npm we know work:
+
+- `npm 3.3.12`
+- `node 5.4.1`
+
+To install gulp:
+
+- run `npm install -g gulp`
```

Comment:
Probably not worth changing it on this PR, and maybe open a separate issue:

Do you think it's better to create npm commands for gulp so that you don't need to install the Gulp package globally and run it from `node_modules`?

Reply:
Gulp it's just for development purpose but it's a good idea. Can you open a separate ticket for that?